apps/io-func-sign-issuer/src/infra/http/handlers/set-signature-request-status.ts

```diff
@@ -0,0 +1,98 @@
+import * as H from "@pagopa/handler-kit";
+
+import { enqueue } from "@io-sign/io-sign/infra/azure/storage/queue";
+import { EventName, createAndSendAnalyticsEvent } from "@io-sign/io-sign/event";
+import * as E from "fp-ts/lib/Either";
+import * as RTE from "fp-ts/lib/ReaderTaskEither";
+
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import { sequenceS } from "fp-ts/lib/Apply";
+
+import { validate } from "@io-sign/io-sign/validation";
+
+import { logErrorAndReturnResponse } from "@io-sign/io-sign/infra/http/utils";
+import { QueueClient } from "@azure/storage-queue";
+import {
+  SetSignatureRequestStatusBody,
+  SetSignatureRequestStatusBodyEnum,
+} from "../../http/models/SetSignatureRequestStatusBody";
+import {
+  SignatureRequest,
+  getSignatureRequest,
+  markAsCancelled,
+  markAsReady,
+  upsertSignatureRequest,
+} from "../../../signature-request";
+import { requireIssuer } from "../../http/decoders/issuer";
+import { requireSignatureRequestId } from "../../http/decoders/signature-request";
+
+const requireSetSignatureRequestStatusBody = (req: H.HttpRequest) =>
+  pipe(
+    req.body,
+    validate(SetSignatureRequestStatusBody),
+    E.filterOrElse(
+      (status) => status === "READY" || status === "CANCELLED",
+      () => new Error("only READY or CANCELLED are allowed")
+    )
+  );
+
+const getQueue =
+  (signatureRequest: SignatureRequest) =>
+  (queueClient: { ready: QueueClient; cancelled: QueueClient }) => {
```

Comment:
Why are there still two queues here if they have been merged into one?

Reply:
ok then i would rename the queue here as well!
```suggestion
  (queueClient: { ready: QueueClient; updated: QueueClient }) => {
```